internal/providers/terraform/aws/vpc_endpoint.go

```diff
@@ -1,128 +1,28 @@
 package aws
 
 import (
-	"fmt"
-	"strings"
-
+	"github.com/infracost/infracost/internal/resources/aws"
 	"github.com/infracost/infracost/internal/schema"
-	"github.com/infracost/infracost/internal/usage"
-	"github.com/shopspring/decimal"
 )
 
-func GetVpcEndpointRegistryItem() *schema.RegistryItem {
+func getVpcEndpointRegistryItem() *schema.RegistryItem {
 	return &schema.RegistryItem{
 		Name:  "aws_vpc_endpoint",
 		RFunc: NewVpcEndpoint,
 	}
 }
-
 func NewVpcEndpoint(d *schema.ResourceData, u *schema.UsageData) *schema.Resource {
-	region := d.Get("region").String()
-	costComponents := []*schema.CostComponent{}
-	vpcEndpointType := "Gateway"
-	vpcEndpointInterfaces := 1
-	var endpointHours, endpointBytes string
-	var gbDataProcessed *decimal.Decimal
-
-	if d.Get("vpc_endpoint_type").Exists() {
-		vpcEndpointType = d.Get("vpc_endpoint_type").String()
-	}
-
-	if len(d.Get("subnet_ids").Array()) > 1 {
-		vpcEndpointInterfaces = len(d.Get("subnet_ids").Array())
-	}
-
-	// Gateway endpoints don't have a cost associated with them
-	if strings.ToLower(vpcEndpointType) == "gateway" {
-		return &schema.Resource{
-			Name:      d.Address,
-			NoPrice:   true,
-			IsSkipped: true,
-		}
-	}
+	subnetIds := d.Get("subnet_ids").Array()
 
-	if u != nil && u.Get("monthly_data_processed_gb").Exists() {
-		gbDataProcessed = decimalPtr(decimal.NewFromFloat(u.Get("monthly_data_processed_gb").Float()))
+	vpcEndpointInterfaces := int64(1)
+	if len(subnetIds) > 0 {
+		vpcEndpointInterfaces = int64(len(subnetIds))
```

Comment:
change to `vpcEndpointInterfacesLen`?

Reply:
As above, have updated to `vpcEndpointInterfaceCount`